react/HistoryRow/styles.styl

```diff
@@ -0,0 +1,37 @@
+@require 'utilities/spaces.styl'
+    
+.HistoryRowCircleWrapper 
+    width: 2.6rem;
+    height: 2.6rem;
+    justify-content: center;
+    display: flex;
+    align-self: start;
+
+.HistoryRowCirle
+    border 1px var(--silver) solid
+    background-color #fff
+
+.HistoryRowRevisionLoader
+    display flex
+    justify-content center
+
+.HistoryRowCaption
+    margin-bottom spacing_values.xl
+    padding-left spacing_values.xl
+    padding-right spacing_values.xl
+
+.HistoryRowMedia
+    position relative
+.HistoryRowMedia::before
+    content ''
+    border-left 1px var(--coolGrey) dashed
+    position absolute
+    margin-left 20px
+    top: 35px
+    bottom: -15px
```

Comment:
Nit: These values seem magical, could you name them to make their purpose more explicit ?